.github/workflows/ci.yml

```diff
@@ -36,16 +27,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Use Node.js latest
-        uses: actions/setup-node@v2-beta
+        uses: actions/setup-node@v3
         with:
           node-version: "*"
+          cache: "yarn"
       - name: 'Check for unmet constraints (fix w/ "yarn constraints --fix")'
         run: |
           yarn constraints
       - name: 'Check for duplicate dependencies (fix w/ "yarn dedupe")'
-        if: steps.yarn-cache.outputs.cache-hit != 'true'
```

Comment:
We do we always run it now?

Reply:
Good catch. I should have left a comment here. This step was previously in the `prepare-yarn-cache` job including the step `yarn-cache`. Not long ago it was extracted to a separate job `yarn-validate` which can't access the `yarn-cache` output anymore, so the condition here is always `true` now. The dedupe check is around 2 seconds so we can just leave it here.